client/src/amplify-ui/cli-feature-flag/feature-flags.json

```diff
@@ -356,6 +356,27 @@
             "defaultExistingProject": true
           }
         ]
+      },
+      "enableDartNullSafety": {
+        "description": "Generate Dart models with null safety for Flutter applications using DataStore. Refer to https://docs.amplify.aws/lib/project-setup/null-safety/q/platform/flutter for more information",
+        "type": "Feature",
+        "valueType": "Boolean",
+        "versionAdded": "5.1.0",
+        "deprecationDate": "Jan 1st 2022",
```

Comment:
What is `deprecationDate` for?

Reply:
Each Feature Flag has a preset deprecation date. Post deprecation, we will no longer generate non-null safe models.